public/css/card.css

```diff
@@ -0,0 +1,24 @@
+
+.card {
+    background: #fff;
```

Comment:
fix indentation, follow the two space indentations. You can format that with vscode